lib/ansible/module_utils/facts.py

```diff
@@ -157,32 +143,32 @@ class Facts(object):
                  { 'path' : '/usr/bin/pkg',         'name' : 'pkg' },
                  { 'path' : '/usr/bin/xbps-install','name' : 'xbps' },
                  { 'path' : '/usr/local/sbin/pkg',  'name' : 'pkgng' },
-    ]
+                ]
 
-    def __init__(self, module, load_on_init=True):
+    def __init__(self, module):
 
         self.module = module
         self.facts = {}
+
         ### TODO: Eventually, these should all get moved to populate().  But
         # some of the values are currently being used by other subclasses (for
         # instance, os_family and distribution).  Have to sort out what to do
         # about those first.
-        if load_on_init:
-            self.get_platform_facts()
```

Comment:
readded it. I didn't see it used when I touched that part, maybe because the branchpoint was too old before the rebase.